File: server/src/main/java/com/doubletrouble/plantynanny/entity/Tree.java
package com.doubletrouble.plantynanny.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Entity
@Getter
@Setter
public class Tree {
    @Id
    @GeneratedValue(strategy = GenerationType.UUID)

    private UUID id;

    private String name;

    private String description;

    private String imageURL;

    private Integer humidity_level;

    private Integer light_hours;
}

File: frontend/app/src/main/java/com/doubletrouble/myapplication/data/model/Tree.kt
package com.doubletrouble.myapplication.data.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Tree(
    val id: String,
    val name: String,
    val description: String,
    val imageURL: String,
    @SerialName("humidity_level") val humidityLevel: Int,
    @SerialName("light_hours") val lightHours: Int
)